src/mame/kaneko/sandscrp.cpp

```diff
@@ -497,7 +498,7 @@ ROM_START( sandscrp ) /* Z03VA-003 PCB */
 	ROM_REGION( 0x20000, "audiocpu", 0 )        /* Z80 Code */
 	ROM_LOAD( "8.ic51", 0x00000, 0x20000, CRC(6f3e9db1) SHA1(06a04fa17f44319986913bff70433510c89e38f1) )
 
-	ROM_REGION( 0x100000, "gfx1", 0 )   /* Sprites */
+	ROM_REGION( 0x100000, "sprites", 0 )   /* Sprites */
```

Comment:
If you name the region "sprites" I think there's no need anymore for the /* Sprites */ comment